pysrc/bytewax/__init__.py

```diff
@@ -5,11 +5,13 @@
 documentation.](https://github.com/bytewax/bytewax)
 
 """
-from .bytewax import AdvanceTo, cluster_main, Dataflow, Emit, run_main
+from .bytewax import AdvanceTo, cluster_main, Dataflow, Emit, KafkaInputConfig, ManualInputConfig, run_main
 from .execution import run, run_cluster, spawn_cluster
 
 __all__ = [
     "Dataflow",
+    "KafkaInputConfig",
+    "ManualInputConfig",
```

Comment:
I couldn't really decide on this. We directly import AdvanceTo and Emit and other "mission critical" kind of pieces of the dataflow under `bytewax`, so that seemed right for the two required configs, but I was thinking the parent struct would stay in `inputs`.  But maybe we should just move the other? Maybe I need to read about philosophies on namespacing? Maybe we need one?